hope.yaml

```diff
@@ -2,14 +2,36 @@ access_points:
   - 192.168.2.43
 access_point_controller: http://192.168.2.10:8080
 nodes:
+  # Hypervisors
+  # This is the list of hypervisors that host each of the VMs described below.
+  - name: beast1
+    role: hypervisor
+    host: 192.168.10.40
+    user: root
+    datastore: Main
+  # Masters
+  # There should be three of these.
   - name: home-master-01
     role: master
     host: 192.168.1.31
     user: root
+  # Nodes
+  # There should be any number of these, primarily dependent on the amount of
+  #   RAM needed to run everything.
+  # Each node should have 8 gigs of memory, but that may change based on what
+  #   hardware is available at the time.
   - name: home-node-01
```

Comment:
Should these even be included anymore? Maybe include some kind of a switch that tells the program to load IPs from the hypervisor, rather than having them written into the config?